### PR TITLE
Removed all Java code and minor refinements in README.md

### PR DESCRIPTION
**Pull request by macemoth**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ This documents defines common Tapas APIs.
 
 ### Task Execution
 
-| TaskList (offering)               |                                          | AuctionHouse (bidding) |
+| TaskList (offering)               |                                          | Roster (bidding party) |
```
Comment (jo9x) [inline README.md]: Not necessarily the Roster right? Think at this point each group needs to decide on their own who does the patch? :)

File: README.md [hunk 2]
```diff
@@ -34,20 +34,22 @@ This documents defines common Tapas APIs.
 
 #### HTTP POST `/auction`
 
-**Media-Type** `application-json`
+**Media-Type** `application-auction+json`
```
Comment (jo9x) [inline README.md]: application/auction+json? (At least thats what i configure in postman dunno if it makes a difference)

Comment (macemoth) [reply README.md]: Thanks :) Rectified in 2faad10

File: README.md (hunk 1 above)
Comment (macemoth) [reply README.md]: Thanks :) Changed in 2faad10

**Pull request merged**